Documentation/button/button.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="../../index.css" />
+    <link rel="stylesheet" href="../../Stylesheets/components/button.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
You can import the components file to main css file and import only that.

Reply:
done!